pkg/key/key.go

```diff
@@ -14,3 +20,47 @@ func GetControlPlaneEndpointHost(clusterName string, baseDomain string) string {
 func ServiceDomain() string {
 	return "cluster.local"
 }
+
+func IgnoreCAPIErrorForField(field string, err error) error {
```

Comment:
God help us

Reply:
Can we move this function to `error.go`?

Reply:
+1